op-service/testutils/mock_eng_deriver.go

```diff
@@ -18,3 +18,7 @@ func (m *MockEngDeriver) TryUpdatePendingSafe(ctx context.Context, ref eth.L2Blo
 func (m *MockEngDeriver) TryUpdateLocalSafe(ctx context.Context, ref eth.L2BlockRef, concluding bool, source eth.L1BlockRef) {
 	m.Mock.MethodCalled("TryUpdateLocalSafe", ctx, ref, concluding, source)
 }
+
+func (m *MockEngDeriver) RequestForkchoiceUpdate(ctx context.Context) {
```

Comment:
I'm not familiar with how mocks are handled in the monorepo yet. If this is built manually, should `MockEngDeriver` be renamed to `MockEngController`?

Reply:
Since `EngDeriver` is removed and the responsibility is embedded to `EngController`, it makes sense to mock `EngController`.